examples/provider/provider-service-account-with-admin-roles.tf

```diff
@@ -0,0 +1,10 @@
+# Auth method: Admin roles applied directly to a service account
```

Comment:
Have you tested this with any resources outside of groups etc? I believe this is only applicable to groups, but I may be wrong. If it is, I think we should make a note that Service Accounts are only able to be granted Groups Admin, in which case they can only manage the groups scope

Reply:
I've had a play around with service accounts given admin roles directly and users can be managed as well as groups -I've made a new acceptance test that demonstrates this but it requires some changes to the acceptance test infra: https://github.com/hashicorp/terraform-provider-googleworkspace/pull/325

I've tested some other things and seen they don't work without impersonation - I'll update the docs with this information!

Reply:
Addressed this in 63173b4 - I think the limitations of using custom admin roles are:
- some resources _must_ be managed by super admins, therefore via user impersonation
- some API endpoints may not be able to be granted via an admin role